src/reward-hooks/index.js

```diff
@@ -1,13 +1,13 @@
 const Rewards = {
   BuriedPawggers: '5aa81e72-5708-434c-ac2e-3dd45e94aa2f',
-  First: '18e618a5-476f-4f63-a353-8c6ff9b5562c',
+  EarlyBirds: '18e618a5-476f-4f63-a353-8c6ff9b5562c',
```

Comment:
Do we need to get a new ID or will it be the same one? Did you create a new reward or update the existing one?